benchmarks/MOABB/hparams/MotorImagery/BNCI2014001/EEGNet.yaml

```diff
@@ -165,3 +215,29 @@ model: !new:models.EEGNet.EEGNet
     dense_max_norm: !ref <dense_max_norm>
     dropout: !ref <dropout>
     dense_n_neurons: !ref <n_classes>
+
+# Search Space
```

Comment:
I like the search space concept!

I wonder if there's a way to reference the search space in the rest of the yaml file -- like:

```
dropout: !ref <search_space.dropout>
```